Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.xaml

```diff
@@ -30,12 +43,12 @@
                                            Margin="0,0,0,2"
                                            FontSize="20"
                                            FontWeight="SemiBold"
-                                           Foreground="{TemplateBinding ValueBrush}"
+                                           Foreground="{TemplateBinding Foreground}"
                                            Text="{TemplateBinding Value}"
                                            TextAlignment="Center" />
                                 <TextBlock Margin="0"
                                            FontSize="16"
-                                           Foreground="{TemplateBinding UnitBrush}"
+                                           Foreground="{ThemeResource RadialGaugeAccentBrush}"
```

Comment:
Can we have this default to the Foreground brush if the `RadialGaugeAccentBrush` isn't specified.

Reply:
How would it not be specified?

Reply:
Foreground could set both UnitBrush and the ValueBrush, while the Accent is transparent, or something like that. Then setting the RadialGaugeAccentBrush can override the UnitBrush color.

Reply:
Sorry... I'm not following. How would RadialGaugeAccentBrush not be specified?

Reply:
As in not manually defined. Probably not important.

Reply:
It's defined in the ThemeDictionary

Reply:
Nvm then